docs/docs/user_guide/secure_install.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Secure Install
+parent: User's Guide to AaC
+nav_order: 1x
+---
+
+# Architecture-as-Code Secure Installation
+In order to support portability of Architecture-as-Code, and to provide a secure, repeatable installation method for AaC users, we have created
+this one-stop installation package for each release of the AaC python package.
```

Comment:
This may not be what you meant but I'm not sure so, just in case...


```suggestion
this one-step installation package for each release of the AaC python package.
```